ui/job-view/pushes/PushJobs.jsx

```diff
@@ -108,7 +108,7 @@ class PushJobs extends React.Component {
     } = this.props;
 
     return (
-      <div role="button" tabIndex={0} onMouseDown={this.onMouseDown}>
+      <div role="list">
```

Comment:
The appropriate fix here would be to keep the `onMouseDown` event handler and remove the role.  But then you'd need to make a listing exception like so:

      // eslint-disable-next-line jsx-a11y/no-static-element-interactions
      <div onMouseDown={this.onMouseDown}>